Learn_CRM_Sdk/CustomViewsViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomViewsViewController.swift
+//  Learn_CRM_Sdk
+//
+//  Created by ZU-IOS on 10/03/20.
+//  Copyright © 2020 ZU-IOS. All rights reserved.
+//
+
+import UIKit
+import ZCRMiOS
+
+protocol customViewDelegate : class{
+    func changeToCustomView(_ leadStatus : String , _ cvRow : Int)
+}
+
+class CustomViewsViewController: UIViewController {
+
+    let customViewTableView : UITableView = UITableView()
```

Comment:
why let for customViewTableView?

Reply:
if no need to change the value
so we using constant